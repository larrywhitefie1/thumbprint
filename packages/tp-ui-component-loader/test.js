const sass = require('node-sass');

it('compiles correctly', () => {
    const css = sass
        .renderSync({ file: 'packages/tp-ui-component-loader/_index.scss' })
        .css.toString();
    expect(css).toMatchSnapshot();
});
