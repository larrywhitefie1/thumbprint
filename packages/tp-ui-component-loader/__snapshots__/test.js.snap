// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
"@keyframes tp-loader-animation-1 {
  0%,
  70%,
  100% {
    opacity: 0.2; }
  30% {
    opacity: 1; } }

@keyframes tp-loader-animation-2 {
  0%,
  15%,
  85%,
  100% {
    opacity: 0.2; }
  45% {
    opacity: 1; } }

@keyframes tp-loader-animation-3 {
  0%,
  30%,
  100% {
    opacity: 0.2; }
  60% {
    opacity: 1; } }

.tp-loader {
  display: flex;
  justify-content: center;
  list-style: none;
  padding: 0;
  margin: 0; }
  .tp-loader > li {
    width: 16px;
    height: 16px;
    border-radius: 50%;
    animation: tp-loader-animation-1 1.3s infinite ease-in;
    background: #009fd9;
    opacity: 0;
    -moz-transform: rotate(0.1deg); }
    .tp-loader > li:nth-child(2) {
      animation-name: tp-loader-animation-2;
      margin-left: 10px;
      margin-right: 10px; }
    .tp-loader > li:nth-child(3) {
      animation-name: tp-loader-animation-3; }
  .tp-loader--small > li {
    width: 8px;
    height: 8px; }
    .tp-loader--small > li:nth-child(2) {
      margin-left: 5px;
      margin-right: 5px; }
  .tp-loader--inverse > li {
    background: #ffffff; }
  .tp-loader--muted > li {
    background: #d3d4d5; }
"
`;
